dependencies/README.md

```diff
@@ -0,0 +1,67 @@
+Static Linkage Checker
+======================
+
+Static Linkage Checker is a tool that finds [static linkage errors](
+../library-best-practices/glossary.md) on a classpath and reports the errors to the console.
+It scans the class files in a classpath provided as input for references
+to other classes.
+It then verifies each of the references to find linkage errors in the classpath.
+The tool also provides analysis on reachability to the errors from entry point
+classes through class usage graphs.
+
+### Use Cases
+
+There are two use cases for Static Linkage Checker:
+
+-  **For organizations** that provide multiple libraries developed by different teams,
+  the tool helps to ensure that there are no static linkage errors among the libraries and their
+  dependencies.
+
+- **For library/application developers** the tool finds static linkage
+  errors in their projects, and will help to avoid incompatible versions of libraries
+  in their dependencies.
+
+### Approach
+
+1. Given an input, the tool prepares a _linkage classpath_, from which the tool inspects Java
```

Comment:
I would say that the required input is a class path. What do you think of this?

1.  The user specifies a class path on which to operate. This is separate from the class path of the tool itself.
2.  The tool lists all _static references_ from classes in the class path that cannot be found in the class path or in the JDK.
3.  Optionally, the user can specify a subset of the class path as "entry points"; in that case the tool will list only those missing references that are _reachable_ from the classes in the entry points.

I think the entry point subset should be specifiable as either a class path or a set of classes, but that's separate.

Reply:
I incorporated the enhancements. Do they look good now?